js/scripts.js

```diff
@@ -0,0 +1,16 @@
+var carouselList = $("#carousel ul");
+	setInterval(changeSlideNext, 2000);
+	
+	function changeSlideNext() {
+		carouselList.animate({'marginLeft': -400}, 1500, moveFirstSlide);
+	};
+	
+	function moveFirstSlide () {
+		var firstItem = carouselList.find("li:first"); // Czy te zmienne można deklarować na początku, 
+		var lastItem = carouselList.find("li:last");// czy trzeba w funkcji?
```

Comment:
tak, bo za każdym razem to mogą być już inne elementy. Chcesz je na nowo przeliczać w momencie odpalenia każdorazowo funkcji.